src/cli/flags.go

```diff
@@ -101,6 +102,12 @@ func InitFlags() flags {
 					Required: true,
 					Category: "deploy",
 				},
+				&cli.StringFlag{
+					Name:     imagePullSecretsFlag,
+					EnvVars:  []string{"INITIUM_REGISTRY_SECRET"},
```

Comment:
As I said during standup I believe we might be better using the same name as the flag for the environment variable:

```suggestion
					EnvVars:  []string{"INITIUM_IMAGE_PULL_SECRET"},
```

I also just realised that this can be an array of elements... We might want to account for this.

Reply:
@LucaLanziani updated the variable name, thanks for the suggestion. Maybe we could handle that array of elements in a separate ticket? I don't think this blocks us